src/main/java/uk/gov/hmcts/reform/unspec/model/UserDetails.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.unspec.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class UserDetails {
```

Comment:
is there not an object from idam dependency for this?

Reply:
Renaming it to UserAuthDetails